package.json

```diff
@@ -7,15 +7,17 @@
   "engines": {
     "node": ">=20.10.0"
   },
+  "type": "module",
   "scripts": {
-    "test": "jest --setupFiles dotenv/config --coverage",
+    "test": "NODE_OPTIONS=\"$NODE_OPTIONS --experimental-vm-modules\" jest --setupFiles dotenv/config --coverage",
```

Comment:
Could you add `cross-env`? I had to to get this to work

Reply:
Sorry I run this directly through `IntelliJ` UI so didn't think of it, will add `crossenv`.